<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewContainerDialog</class>
 <widget class="QDialog" name="NewContainerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name of container</string>
   </property>
  </widget>
  <widget class="QLabel" name="urlLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>HTTP of FTP address of image</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>491</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="urlEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>491</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loadUrlButton">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>140</y>
     <width>215</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Load image by HTTP or FTP</string>
   </property>
  </widget>
  <widget class="QLabel" name="urlError">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>144</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>69</red>
         <green>69</green>
         <blue>69</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>69</red>
         <green>69</green>
         <blue>69</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="localEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>401</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="localLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Local address of image</string>
   </property>
  </widget>
  <widget class="QLabel" name="localError">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>254</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>69</red>
         <green>69</green>
         <blue>69</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>69</red>
         <green>69</green>
         <blue>69</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="loadLocalButton">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>250</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Install image from local disk</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseButton">
   <property name="geometry">
    <rect>
     <x>426</x>
     <y>216</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
